Rakefile

```diff
@@ -5,4 +5,4 @@ require File.expand_path("config/application", __dir__)
 
 ManualsFrontend::Application.load_tasks
 
-task default: ["jasmine:ci"]
+task default: [:lint, :spec, "jasmine:ci"]
```

Comment:
So this is a problem I have with the way RSpec integrates with Rake. We should:

- Apply the approach [here](https://github.com/alphagov/content-publisher/blob/master/Rakefile#L7) to other PRs around this
- Make a consistent decision about whether `lint` comes before/after `spec` (about 50/50) at the moment

Reply:
I think it should come before. But it is a bit annoying when you just want to see if a test passes on your WIP commit...

Reply:
```suggestion
Rake::Task[:default].clear
task default: [:lint, :spec, "jasmine:ci"]
```